### PR TITLE
Add overview section

### PR DESCRIPTION
**Pull request by mnabywan**
Move overview from Guides to separate section and separate  'What is EPICS' from Introduction.

### CODE REVIEW

File: overview/What_Is_EPICS.rst [hunk 1]
```diff
@@ -0,0 +1,56 @@
+What is EPICS?
+--------------
+
+.. image:: ../images/EPICS_black_blue_logo_rgb_small_v03.jpg
```
Comment (gabrielfedel) [inline overview/What_Is_EPICS.rst:4]: I tried to build and this message wasn't found. Is that right ?

**Comment by gabrielfedel** [comment]
Besides the above comment, LGTM!

File: overview/What_Is_EPICS.rst (hunk 1 above)
Comment (minijackson) [reply overview/What_Is_EPICS.rst:4]: Image also doesn't show up in the built website: https://epics--23.org.readthedocs.build/en/23/overview/What_Is_EPICS.html

Comment (mnabywan) [reply overview/What_Is_EPICS.rst:4]: Should be found, I'm checking

**Comment by mnabywan** [comment]
There were quite lot of changes on master, I close that PR and reopen it here: https://github.com/epics-docs/epics-docs/pull/34

**Pull request closed without merging**